go/src/socialapi/workers/api/modules/participant/participant.go

```diff
@@ -45,22 +47,51 @@ func List(u *url.URL, h http.Header, _ interface{}, context *models.Context) (in
 func AddMulti(u *url.URL, h http.Header, participants []*models.ChannelParticipant, context *models.Context) (int, http.Header, interface{}, error) {
 	query := context.OverrideQuery(request.GetQuery(u))
 
+	bo := backoff.NewExponentialBackOff()
+	bo.InitialInterval = time.Millisecond * 50
+	bo.MaxInterval = time.Millisecond * 100
+	bo.MaxElapsedTime = time.Second * 2
```

Comment:
NewTicker handles that https://github.com/cenkalti/backoff/blob/8edc80b07f38c27352fb186d971c628a6c32552b/ticker.go#L45